### PR TITLE
Delete item by item_id

### PR DESCRIPTION
**Pull request by yogeshnile**
Added a new feature to delete an item by item_id on monday.

### CODE REVIEW

**Review by chdastolfo — CHANGES_REQUESTED** [review]
I realize these weren't present when you opened this PR, but since they are now, could you please pull down the latest changes from this repo and add tests for this change? Thanks.

**Comment by yogeshnile** [comment]
I couldn't understand which file you want to change. please explain

**Comment by chdastolfo** [comment]
You need to add at least one unit test to the `test_item_resource.py` file if you want this change to be considered. If you pull the latest changes on the master branch onto your branch, you should see the file I'm talking about.

File: monday/query_joins.py [hunk 1]
```diff
@@ -456,3 +456,14 @@ def get_complexity_query():
     }'''
 
     return query
+
+def delete_item_query(item_id):
```
Comment (chdastolfo) [inline monday/query_joins.py:460]: Please move this query to the section for Item Resource Queries at the top of the file.

File: monday/resources/items.py [hunk 2]
```diff
@@ -38,3 +38,7 @@ def change_multiple_column_values(self, board_id, item_id, column_values):
     def add_file_to_column(self, item_id, column_id, file):
         query = add_file_to_column_query(item_id, column_id)
         return self.file_upload_client.execute(query, variables={'file': file})
+    
+    def delete_item_by_id(self,item_id):
```
Comment (chdastolfo) [inline monday/resources/items.py:42]: Add a space after the comma.

**Review by chdastolfo — CHANGES_REQUESTED** [review]

File: monday/query_joins.py (hunk 1 above)
Comment (yogeshnile) [reply monday/query_joins.py]: file updated

File: monday/resources/items.py (hunk 2 above)
Comment (yogeshnile) [reply monday/resources/items.py]: file updated

**Review by chdastolfo — APPROVED** [review]

**Comment by rhymiz** [comment]
@all-contributors please add @yogeshnile for code

**Pull request merged**